libnmstate/nm/plugin.py

```diff
@@ -140,6 +144,18 @@ def get_dns_client_config(self):
     def refresh_content(self):
         self._ctx.refresh_content()
 
+    @staticmethod
+    def check_version_mismatch():
+        nm_client_version = _nm_client_decode_version()
+        nm_utils_version = _nm_utils_decode_version()
+
+        if nm_client_version is None:
+            warn("NetworkManager is not running\n")
+        elif StrictVersion(nm_client_version) != StrictVersion(
+            nm_utils_version
+        ):
+            warn("Version mismatch\n")
```

Comment:
It would be good to have a log like "libnm version XXXX mismatches NetworkManager version YYYY"

Reply:
Done, thank you!